.circleci/config.yml

```diff
@@ -83,12 +85,11 @@ commands:
   deploy-pip-snapshot-unix:
     steps:
       - run: |
-          ulimit -n 100000
```

Comment:
Note: CircleCI docker is not privileged and cannot set ulimit. We may have to split linux and mac jobs if this proves an issue.